### DISCUSSION TITLE
When I send an email using emailoperator, how can I get the task id that triggered it?

### DISCUSSION THREAD

**Discussion by YaoJusheng** [apache/airflow#24175] [Q&A]
For example, I have a dag that sets up a series of task nodes. 
When any node task fails, an email alert will be triggered. 
I want to display the task ID that triggered the alert and the log link related to the task in the email content, 
so I should How to do it? 
I need help, thanks a lot!

**Comment by potiuk** [comment]
You can extend Email operator and get it the id from the context of execute method.

**Reply by YaoJusheng** [reply]
okay, I have solved it.
